src/dcqc/file.py

```diff
@@ -101,6 +100,7 @@ def get_file_type(cls, file_type: str) -> FileType:
 FileType("OME-TIFF", (".ome.tif", ".ome.tiff"), "format_3727")
 
 
+# TODO: Leverage post-init function in dataclasses
```

Comment:
At some point, all of the todos should be translated to Jira tickets so that the work can be shared across dpe.

Reply:
Most of them don't need to be converted into tickets. Many of these are reminder to refactor bits the next time I touch that code. I would argue that creating tickets will make it less likely for these to get done because everything else will always be higher priority. Instead, I prefer that we be constantly vigilant for improvement opportunities and conduct these improvements as we work on user-driven tickets. You'll come across this point in the Clean Coder book.